scripts/scil_flip_streamlines.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Flip tracts around specific axes.
```

Comment:
Since this script is now called `flip_streamlines`, please change `tracts` to `streamlines` or `tractogram`.